packages/libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -857,61 +857,6 @@ export class DiscoveryProvider {
     })
     return await this._makeRequest(req)
   }
-
-  /**
-   * @deprecated Migrate to using getBestNewReleases
-   */
-  async getTopFolloweeWindowed(
```

Comment:
> ok to delete unused fns in [DiscoveryProvider.ts](https://github.com/AudiusProject/audius-protocol/compare/rt-explore-tracks?expand=1#diff-c9ef1a2c1cb40b6896031fec078f420653e5c016598145f8a31ecef444326b94) and [requests.ts](https://github.com/AudiusProject/audius-protocol/compare/rt-explore-tracks?expand=1#diff-55ed2d5316c983125b559025435418a3e12e6da25c3bf59f7bee4be22432fde7)?

I have generally _not_ been removing things from libs, since it's still used in other packages and it's hard to be sure it won't break things (i.e. JS code that won't throw type errors). It doesn't hurt anything to leave it in, since we will be removing it from client entirely. And once everything is migrated off of libs, we'll just delete it.

However please _do_ remove things from AudiusBackendInstance and anything above libs if they are unused.